### PR TITLE
Add a search feature to the listCourtPage

### PR DESCRIPTION
**Pull request by Alexander-Schulte**
Hello again! :) Just a small PR, implemented a search feature to the listCourtPage. So now the user can filter by name and location to find his perfect court! :)
Thanks for your review! :)

### CODE REVIEW

File: src/components/ListCourtPage.js [hunk 1]
```diff
@@ -1,16 +1,23 @@
...
+import { getItemsFromLocalStorage } from "../services/CourtStorage";
+
 import { useEffect, useState } from "react";
 import "./ListCourtPage.css";
 import CourtCard from "./CourtCard";
+import SearchInListPage from "./SearchInListPage";
 
 function ListCourtPage() {
   const [courtData, setCourtData] = useState([]);
+  const [nameFilter, setNameFilter] = useState("");
```
Comment (FriederikeM) [inline src/components/ListCourtPage.js:10]: Maybe now that you are filtering for both, the name and the location, you can rename this useState

**Review by FriederikeM — APPROVED** [review]
LGTM :) one mini comment 💃

File: src/components/AddCourtPage.js [hunk 2]
```diff
@@ -1,5 +1,5 @@
 import "./AddCourtPage.css";
-import { addItemToLocalStorage } from "./services/CourtStorage";
+import { addItemToLocalStorage } from "../services/CourtStorage";
```
Comment (marwinburesch) [inline src/components/AddCourtPage.js:2]: Would've expected this in another PR

File: src/components/CourtCard.js [hunk 3]
```diff
@@ -9,7 +9,11 @@ function CourtCard({ court, index, onClick, className }) {
     onClick(index);
   }
   return (
-    <article className={className} onClick={handleCourtCardClicked} key={index}>
+    <article
+      className={className}
+      onClick={handleCourtCardClicked}
+      key={court.location}
```
Comment (marwinburesch) [inline src/components/CourtCard.js:15]: Please consider using an `id` in the future, just to make sure it is unique. Maybe you'll have dumb users :)

File: src/components/ListCourtPage.js [hunk 4]
```diff
@@ -34,7 +51,15 @@ function ListCourtPage() {
       );
     });
   }
-  return <div>{renderCourt()}</div>;
+  return (
+    <div>
+      <div className="listHeaderWrapper">
```
Comment (marwinburesch) [inline src/components/ListCourtPage.js:56]: Could be a `<header>`

File: src/components/ListCourtPage.js [hunk 5]
```diff
@@ -34,7 +51,15 @@ function ListCourtPage() {
...
+        <h2 className="listCourtHeader">Court List</h2>
+        <SearchInListPage onFilterNameChange={handleSearchFilterName} />
+      </div>
+      <div className="listContentWrapper">{renderCourt()}</div>
```
Comment (marwinburesch) [inline src/components/ListCourtPage.js:60]: Could be a `<section>`, right?

File: src/components/Map.js [hunk 6]
```diff
@@ -1,4 +1,4 @@
-import { getItemsFromLocalStorage } from "./services/CourtStorage";
+import { getItemsFromLocalStorage } from "../services/CourtStorage";
```
Comment (marwinburesch) [inline src/components/Map.js:1]: Is `Map.js` part of the search?

**Review by marwinburesch — CHANGES_REQUESTED** [review]
Nice and concise :), good work 🐜 

Cheers 🍻

**Pull request merged**